tensorflow-core-ops/Setup.hs

```diff
@@ -81,7 +84,11 @@ fudgePackageDesc lbi p = p
     }
   where
     fudgeBuildInfo bi =
+#if MIN_VERSION_Cabal(3,0,0)
+        bi { hsSourceDirs = unsafeMakeSymbolicPath (autogenModulesDir lbi) : hsSourceDirs bi }
```

Comment:
it is odd that there isn't an obvious way to create a `SymbolicPath` except by using this "unsafe" function

Reply:
I agree. Looked for a while but didn't see any other way to do it.